docs/notebooks/trust_region.pct.py

```diff
@@ -245,6 +245,44 @@ def plot_history(result: trieste.bayesian_optimizer.OptimizationResult) -> None:
 # %%
 plot_history(result)
 
+# %% [markdown]
+# ## TEST
```

Comment:
(?)

Reply:
This is just a section to show how you can use local models. I mentioned in the PR description that I was going to delete it before merging. However, since the TURBO PR now shows how to use this as well, I'll delete it now.